Code/Tools/TestImpactFramework/Runtime/Code/Source/Artifact/Factory/TestImpactTestRunSuiteFactory.cpp

```diff
@@ -69,6 +69,7 @@ namespace TestImpact
                     const auto getDuration = [Keys](const AZ::rapidxml::xml_node<>* node)
                     AZ_POP_DISABLE_WARNING
                     {
+                        AZ_UNUSED(Keys); // Clang reports a warning that capturing Keys is not necessary because it is not odr-used
```

Comment:
This can be avoided by assigning the Keys constant to a different name capture variable
```
[keys = Keys](const AZ::rapidxml::xml_node<>* node)
```